### PR TITLE
First step in dding functionality to be able to filter on the list it…

### PR DESCRIPTION
**Pull request by EdwinGuzman**
…em cards.

This is the first step in filtering and didn't want to do more so this PR wouldn't get too big. It's currently an "or" filtering which was mostly accidental. The next iteration will be an "and" filtering. Removing the filters as you select them is a whole different, huge task, so I'll work on that after this first logic. 

@ricardoom :
screen shot 2017-10-24 at 9 14 15 am

### CODE REVIEW

**Comment by ricardoom** [comment]
@EdwinGuzman good work... I'll ping you & @rhernand3z on whats next..

File: src/app/components/BookFilters/Filter.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }, 1000);
+  }
+
+  render() {
+    const {
+      selected,
+      buttonActive,
+    } = this.state;
+    const filter = this.props.filter;
+    const active = selected ? 'active' : '';
```
Comment (rhernand3z) [inline src/app/components/BookFilters/Filter.jsx]: I would give it `activeClass` name to be clearer it's a class.

**Comment by EdwinGuzman** [comment]
Enzyme, chai, mocha. I want to write tests, but deadline 💀

File: src/app/components/BookFilters/Filter.jsx (hunk 1 above)
Comment (EdwinGuzman) [reply src/app/components/BookFilters/Filter.jsx]: 👌

**Pull request merged**